### PR TITLE
EdgeMesh: remove docker0 dependence

### PR DESCRIPTION
**Pull request by daixiang0**
**What this PR does / why we need it**:

Now EdgeMesh bind docker0 to work as a DNS server, this pr is to remove this dependence.

**Special notes for your reviewer**:

The proxy part of EdgeMesh still depends on docker0 network interface, which would be optimized after related proposal confirmed.

### CODE REVIEW

File: edgemesh/pkg/dns/dns.go [hunk 1]
```diff
@@ -77,29 +77,47 @@ var metaClient client.CoreInterface
 // dnsConn saves DNS protocol
 var dnsConn *net.UDPConn
 
+//Init creates network interface
+func Init(addr string) {
+	// create a network interface
+	checkInterface := fmt.Sprintf("brctl show | grep %s 2>/dev/null", ifi)
```
Comment (jimmy-wei) [inline edgemesh/pkg/dns/dns.go:83]: Is it possible to use [netlink ](https://github.com/vishvananda/netlink) instead of system command?

**Comment by Poorunga** [comment]
@daixiang0 This is a great PR, would you like to move this design to the edgemesh new repo: https://github.com/kubeedge/edgemesh

**Pull request closed without merging**